lib/queries/getIdentity.ts

```diff
@@ -0,0 +1,38 @@
+import { FetchResult } from "apollo-link";
+import gql from "graphql-tag";
+
+import type { ProviderUser } from "@src/@types/ProviderUser";
+import { fetchManagement } from "@src/utils/fetchManagement";
+
+const USER_QUERY = gql`
```

Comment:
On this subject, the query document in `getIdentities` should have been called `GET_USER_IDENTITIES_QUERY`, so this one should be

```suggestion
const GET_USER_IDENTITY_QUERY = gql`
```

Could you also change the name in `getIdentities`? Thanks!

Reply:
OK fixed 7714de7

Reply:
OK fixed 69a9c93